_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+* Add support for `json` type.
+* Support reading MongoDB `DBRef` type.
+* Allow skipping creation of an index for the `_schema` collection if it already
+exists.
+* Added support to redact the value of `mongodb.credentials` in the server log.
+* Added support for dropping columns.
+
+### MySQL connector
+
+* Added support for reading and writing `timestamp` values with precision higher than 3.
+* Added support for predicate pushdown on `timestamp` columns.
+* Exclude an internal `sys` schema from schema listings.
+
+### Pinot connector
+
+* Updated Pinot connector to be compatible with versions >= 0.8.0 and drop 
+support for older versions.
+* Added support for pushdown of filters on `varbinary` columns to Pinot.
+* Fixed incorrect results for queries that contain aggregations and `IN` and 
+`NOT IN` filters over varchar columns.
+* Fixed failure for queries with filters on `real` or `double` columns having 
+`+Infinity` or `-Infinity` values.
+* Implemented aggregation pushdown for `count`, `avg`, `min`, `max`, `sum`, 
+`count(DISTINCT)` and `approx_distinct`. It is enabled by default and can be 
+disabled using the configuration property `pinot.aggregation-pushdown.enabled` 
+or the catalog session property `aggregation_pushdown_enabled`.
+* Allowed https URLs in `pinot.controller-urls`. 
+
+## Runtime Improvements
+
+* Improved CPU coordinator utilization.
+* Improved query performance by reducing CPU overhead of repartitioning data 
+across worker nodes.
+* Reduced graceful shutdown time for worker nodes.
+
+## Everything else
+
+* Added support for ARM64 in the Trino Docker image.
+* Added cancelling a Trino query execution when JDBC statement is closed.
+* Added `clear` command to the Trino CLI to clear the screen.
+
+
+## Trino Summit
+
+This year as with the name change, we had the inaugural Trino Summit hosted by
+Starburst and had well over 500 attendees. There were wonderful talks
+given at this event from companies like, Netflix, LinkedIn, EA, Robinhood, 
+Doordash, Tabular, and Stream Native. If you missed this event, we have the 
+[recordings and slides available here](https://www.starburst.io/resources/trino-summit/).
+
+As a teaser, the event started with Commander Bun Bun playing guitar to AC/DC's,
+"Back In Black".
+
+<iframe src="https://www.youtube.com/embed/c_qUp0SGeKE"  
+width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" 
+scrolling="no" style="border:1px solid #CCC; border-width:1px; 
+margin-bottom:5px; max-width: 100%;" allowfullscreen> 
+</iframe>
+
+## 2022 Roadmap: Project Tardigrade
+
+One of the interesting developments that came out of Trino Summit was a feature
+Trino co-creator, Martin, talked about in [the State of Trino presentation](https://www.starburst.io/resources/trino-summit/?wchannelid=2ug6mgs5ao&wmediaid=o264qw85dj).
+He discussed a prioritization in granular fault tolerance and performance in the
+core engine. While Trino has been proven at scale to run batch processes, many
```

Comment:
Weird sentence... maybe better to say its proven to support analytics workloads at scale .... now we are working towards batch processing including long running queries..